tests/scripts/test_stdlib_call_audit.py

```diff
@@ -35,3 +46,24 @@ def test_no_encoding():
     cmd = ['echo', '-n', '-e', '\\xeb']
     result = run(cmd, encoding=None)
     assert isinstance(result['stdout'], six.binary_type)
+
+
+def test_missing_executable(monkeypatch, adjust_cwd):  # no-qa: W0613; pylint: disable=unused-argument
```

Comment:
Hmm are you actually using adjust_cwd? Can't follow the magic

Reply:
it's magic for me too. the same is used in unit-tests for leapp actors in this way. And I remember it replaced the original functionality. so it's either nonsense everywhere or it works somehow magically. I have to admit that dropping it I do not see any difference in test results here. I remember it was explained to me that just having this in the parameters is automagically used already by the pytest because it's decorated by `pytest.fixture`.

Reply:
ah, ok, so it really works. it's pytest magic around fixtures: https://docs.pytest.org/en/6.2.x/fixture.html#teardown-cleanup-aka-fixture-finalization